src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -888,16 +888,16 @@ public function set_status( $status ) {
 		}
 
 
-		$status = wp_update_post( array(
-			'ID' => $this->id,
-			'post_status' => Tribe__Events__Aggregator__Records::$status->{ $status },
-		) );
+		$status = wp_update_post( [
+			'ID'          => $this->id,
+			'post_status' => Tribe__Events__Aggregator__Records::$status->{$status},
+		] );
 
 		if ( ! is_wp_error( $status ) && ! empty( $this->post->post_parent ) ) {
-			$status = wp_update_post( array(
-				'ID' => $this->post->post_parent,
+			$status = wp_update_post( [
+				'ID'            => $this->post->post_parent,
 				'post_modified' => date( Tribe__Date_Utils::DBDATETIMEFORMAT, current_time( 'timestamp' ) ),
-			) );
+			] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: d9dfbd61a671380bc99eb97b4e5e06e9</sub>